tools/c7n_azure/tests/test_app_service_plan.py

```diff
@@ -42,3 +47,58 @@ def test_find_by_name(self):
         })
         resources = p.run()
         self.assertEqual(len(resources), 1)
+
+    @arm_template('appserviceplan.json')
+    def test_resize_plan(self):
+        p = self.load_policy({
+            'name': 'test-azure-appserviceplan',
+            'resource': 'azure.appserviceplan',
+            'filters': [
+                {'type': 'value',
+                 'key': 'name',
+                 'op': 'eq',
+                 'value_type': 'normalize',
+                 'value': 'cctest-appserviceplan'}],
+            'actions': [
```

Comment:
Can you add this schema to the schema validate test too?

Reply:
Like just add the new actions. It'll catch if someone edits that action to create a session in init.